### DISCUSSION TITLE
Building on MAC OSX

### DISCUSSION THREAD

**Discussion by cbailey107** [szpajder/dumpvdl2#30] [Q&A] [answered]
Does anyone have a starting document for building on OSX?  Having issues with library paths.
Thanks 
Chris Bailey

**Comment by szpajder** [comment, marked as answer]
Would the Github CI/CD script be of any help?

https://github.com/szpajder/dumpvdl2/blob/master/.github/workflows/build.yml

It manages to build the program successfully on MacOS.

**Comment by cbailey107** [comment]
I will give it shot.  One of the issues is update_dyld_shared_cache is deprecated so its not finding the libacars-2 library.

**Reply by cbailey107** [reply]
build was successful. will do some testing over the next week or 2.
